util/p9/bkreplace

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+timestamp=$(date +"%Y-%m-%d_%H-%M-%S")
+
+echo "backing up gatling-puppet-load-test"
+tar czf ./gplt.${timestamp}.tar.gz ./gatling-puppet-load-test
+
+echo "removing gatling-puppet-load-test"
+rm -rf ./gatling-puppet-load-test
+
+echo "cloning gatling-puppet-load-test"
+git clone https://github.com/puppetlabs/gatling-puppet-load-test.git
+
+echo "refresh complete; adding timestamp to gatling-puppet-load-test"
+touch ./gatling-puppet-load-test/${timestamp}.txt
```

Comment:
If a target destination path is added above, this line will need to be updated.

Reply:
TDB whether a target destination path should be added. Added [docs/slv_environment_setup.md](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md) with a [Utility scripts](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md#utility-scripts) section.